04_HowTos/SPARCL/How_to_use_SPARCL.html

```diff
@@ -7588,6 +7588,11 @@ <h2 id="Goals-&amp;-Summary">Goals &amp; Summary<a class="anchor-link" href="#Go
 <li><a href="https://github.com/astro-datalab/notebooks-latest/blob/master/03_ScienceExamples/EmLineGalaxies/01_EmLineGalaxies_SpectraStack.ipynb">Stacking SDSS Spectra of Galaxies Selected from the BPT Diagram</a></li>
 <li><a href="https://github.com/astro-datalab/notebooks-latest/blob/master/03_ScienceExamples/EmLineGalaxies/02_EmLineGalaxies_Outliers.ipynb">Multi-wavelength Image Cutouts and SDSS Spectra of Active Galaxies with Extreme Emission-Line Ratios</a></li>
 </ul>
+<p>And our other How-To notebooks that use SPARCL:</p>
+<ul>
+<li><a href="https://github.com/astro-datalab/notebooks-latest/blob/master/04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Jdaviz.ipynb">SPARCL + Jdaviz</a></li>
+<li><a href="https://github.com/astro-datalab/notebooks-latest/blob/master/04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Prospect.ipynb">Obtain and plot spectra data using SPARCL, prospect, and specutils</a></li>
```

Comment:
I suggest making the notebook short descriptions more homogenous, such as: "Obtain spectra with SPARCL and plot them with Jdaviz" and "Obtain spectra with SPARCL and plot them with prospect" (I think specutils is an intermediary step for converting the data format).